### PR TITLE
CLC-5136, rake task and shell scrpt in support of Webcast cron job

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/CLC-5136

The rake task will be implemented per https://jira.ets.berkeley.edu/jira/browse/CLC-5242.

### CODE REVIEW

**Comment by raydavis** [comment]
Travis is half full.

File: script/webcast.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+[[ -s "${HOME}/.rvm/scripts/rvm" ]] && . "${HOME}/.rvm/scripts/rvm"
+source .rvmrc
+
+export RAILS_ENV=production
+export LOGGER_STDOUT=only
+export LOGGER_LEVEL=INFO
+
+echo | $LOGIT
+echo "------------------------------------------" | $LOGIT
+echo "`date`: Webcast cron job started on app node: `hostname -s`..." | $LOGIT
```
Comment (raydavis) [inline script/webcast.sh]: "Webcast cron job" doesn't describe what the script is doing. How about "Webcast LTI app visibility checks" or something like that?

**Comment by raydavis** [comment]
Another naming request, @johncrossman -- "webcasts.sh" seems like too general a filename to convey at a glance what the script is for. We have a lot of script files, so it's probably worth being more specific so that we can quickly spot the one which checks to see if bCourses webcast apps should be unhidden.

**Comment by johncrossman** [comment]
Thanks @raydavis  I've done some renaming

**Comment by raydavis** [comment]
Thank you, @johncrossman !

**Pull request merged**